src/utils/logger.utils.ts

```diff
@@ -0,0 +1,35 @@
+import { METICULOUS_LOGGER_NAME } from "@alwaysmeticulous/common";
+import log from "loglevel";
+
+export const initLogger: () => void = () => {
```

Comment:
These are exported from CLI, but think it's better just to copy and paste them and kill the CLI dep (not complex enough to be worth their own package)